systemd/ceph-radosgw@.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Ceph rados gateway
+After=network-online.target local-fs.target
+Wants=network-online.target local-fs.target
+PartOf=ceph.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/ceph
+Environment=CLUSTER=ceph
+ExecStart=/usr/bin/radosgw -f --conf /etc/ceph/${CLUSTER}.conf --name client.radosgw.%i
```

Comment:
sure -> I am happy with this substitution.

I felt it was cleaner to have ${CLUSTER} as part of the environment, are you saying that if the environment variable is specified  "--conf /etc/ceph/${CLUSTER}.conf" is not needed?